# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RendererOGL.h
# Commit: 91d8f0f829fd
# Full Hash: 91d8f0f829fd2ae6ec234a6a3f372bf503ed2fbd
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-06 04:14:34
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add RenderTextureHostSWGL support to D3D11 RenderTextureHosts. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95799
# ==============================================================================

diff -r 91cb8ee8ddd8 -r 91d8f0f829fd gfx/webrender_bindings/RendererOGL.h
--- a/gfx/webrender_bindings/RendererOGL.h	Thu Nov 05 19:26:21 2020 +0000
+++ b/gfx/webrender_bindings/RendererOGL.h	Thu Nov 05 20:18:13 2020 +0000
@@ -113,6 +113,8 @@
 
   RenderTextureHost* GetRenderTexture(wr::ExternalImageId aExternalImageId);
 
+  RenderCompositor* GetCompositor() { return mCompositor.get(); }
+
   void AccumulateMemoryReport(MemoryReport* aReport);
 
   void SetProfilerUI(const nsCString& aUI);
